### PR TITLE
added tailwind, styled/tw-styled components packages and navbuttons component

### PR DESCRIPTION
**Pull request by Steviemaru**

### CODE REVIEW

File: app/components/NavButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    py-1
+    px-7
+    w-32  
+  `;
+
+  return (
+    <>
+      <Wrapper>
+       {navItems.map((item) => {
+        console.log(item, selectedItem, "check");
```
Comment (yoloOnTheBattlefield) [inline app/components/NavButtons.tsx]: no console

File: app/components/NavButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  };
+const width = styled.div`
+width:320px;
+`
+  const Wrapper = tw(width)`
+  flex
+  bg-black
+  bg-opacity-50
+  rounded-3xl  
+`;
```
Comment (yoloOnTheBattlefield) [inline app/components/NavButtons.tsx:12]: don;t put the styled components in the component

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/NavButtons.tsx (hunk 1 above)
Comment (Steviemaru) [reply app/components/NavButtons.tsx]: sorry forgot to "git add" the changes for this.

File: app/components/NavButtons.tsx (hunk 2 above)
Comment (Steviemaru) [reply app/components/NavButtons.tsx:12]: oh do you mean the function description? should I declare this outside of the Navbuttons component ?

File: app/components/NavButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    m-3
+    py-1
+    px-7
+    w-32  
+  `;
+
+export default function NavButtons() {
+  const [navItems] = useState(["coins", "portfolio"]);
+  // need this to toggle background color on button component
+  // will use this after I install redux and setup provider
```
Comment (yoloOnTheBattlefield) [inline app/components/NavButtons.tsx:35]: ok

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**